src/ORM/Connect/MySQLDatabase.php

```diff
@@ -367,14 +367,12 @@ public function transactionDepth()
 
     public function transactionEnd($chain = false)
     {
-        $result = $this->getTransactionManager()->transactionEnd();
-
         if ($chain) {
-            Deprecation::notice('4.4', '$chain argument is deprecated');
+            Deprecation::notice('5.0.0', 'The $chain parameter will be removed in CMS 5');
```

Comment:
```suggestion
            Deprecation::notice('4.4.0', 'The $chain parameter for ' . __METHOD__ . ' is deprecated', Deprecation:SCOPE_GLOBAL);
```
The old version is correct - the version should be when it was deprecated, not when it's being removed.

The new message should say what method it's in, and it shouldn't declare when it will be removed. In this case we do know it has actually been removed in CMS 5 but as a rule we have decided deprecation notices should avoid saying when the API is removed for consistency. 

Global scope is needed so we don't have a message saying the method is deprecated, since it's not the whole method that's deprecated.

Reply:
DONE

Reply:
Close enough 👌